mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -193,12 +207,19 @@ class KMeans private (
 
     val initStartTime = System.nanoTime()
 
-    val centers = if (initializationMode == KMeans.RANDOM) {
-      initRandom(data)
-    } else {
-      initKMeansParallel(data)
+    val centers = initialModel match {
+      case Some(kMeansCenters) => {
+        Array.tabulate(runs)(r => kMeansCenters.clusterCenters
```

Comment:
@jkbradley Irrespective of the value of `runs` set by the user, do you mean to reassign `runs` to 1 from the setInitialModel()  along with a warning message ?
To document this behavior , can we do it in this PR itself  or should we start a new ticket for documentation?